### DISCUSSION TITLE
Junior coaster allows for steep drops in rct1

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17322] [Feature requests] [answered]
Could you add steep drops to openrct2 junior coaster from rct1 junior coaster?

**Comment by spacek531** [comment, marked as answer]
This already exists in the form of the Classic Mini Steel Coaster.

**Comment by Gymnasiast** [comment]
It should also be noted that RCT1’s Steel Mini RollerCoaster is a strange amalgamation of the (Zierer) Junior Coaster and (Schwarzkopf) Mini Coaster. RCT2 is actually correct in this regard and RCT1 is wrong. Nevertheless, we are reintroducing all the RCT1 features, under the name of "Classic Mini Roller Coaster". At the moment, we are working on trains to port.
